client/me/purchases/manage-purchase/index.jsx

```diff
@@ -353,7 +360,7 @@ const ManagePurchase = React.createClass( {
 			</span>
 		);
 
-		if ( isLoading || ! showEditPaymentDetails( purchase ) ) {
+		if ( isLoading || ! showPaymentDetails( purchase ) ) {
```

Comment:
Super minor, but I think it'd be better to move the inside `showPaymentDetails` to here and to get rid of that function entirely, for a couple reasons:

- That function is only used once, and it's slightly easier to read this method if we don't have to look up another function.
- If we keep `showPaymentDetails`, we might want to rename it (it doesn't actually show payment details).

The condition would look like this:

```js
if ( isLoading || ! canEditPaymentDetails( purchase ) || ! isPaidWithCreditCard( purchase ) ) {
```